src/Components/JobsContainer/JobsContainer.tsx

```diff
@@ -1,56 +1,67 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import {
-    IJobsWrapperProps,
-    IJobsWrapperStyleProps,
-    IJobsWrapperStyles
-} from './JobsWrapper.types';
-import { getStyles } from './JobsWrapper.styles';
+    IJobsContainerProps,
+    IJobsContainerStyleProps,
+    IJobsContainerStyles
+} from './JobsContainer.types';
+import { getStyles } from './JobsContainer.styles';
 import { classNamesFunction, CommandBar, styled } from '@fluentui/react';
 import { useExtendedTheme } from '../../Models/Hooks/useExtendedTheme';
 import JobsList from './Internal/JobsList/JobsList';
 import JobsDialog from './Internal/JobsDialog/JobsDialog';
 import useAdapter from '../../Models/Hooks/useAdapter';
-import { AdapterMethodParamsForJobs, IJob } from '../../Models/Constants';
+import { AdapterMethodParamsForJobs, IAdtApiJob } from '../../Models/Constants';
 
 const getClassNames = classNamesFunction<
-    IJobsWrapperStyleProps,
-    IJobsWrapperStyles
+    IJobsContainerStyleProps,
+    IJobsContainerStyles
 >();
 
-const JobsWrapper: React.FC<IJobsWrapperProps> = ({
-    adapter,
-    listOfJobs,
-    styles
-}) => {
+const JobsContainer: React.FC<IJobsContainerProps> = ({ adapter, styles }) => {
     // contexts
 
     // state
     const [isJobsDialogOpen, setIsJobsDialogOpen] = useState(false);
+    const [listOfJobs, setListOfJobs] = useState<Array<IAdtApiJob>>([]);
 
-    // hooks
-    const deleteJob = useAdapter({
-        adapterMethod: (params: AdapterMethodParamsForJobs) =>
-            adapter.deleteJob(params),
+    const getAllJobs = useAdapter({
+        adapterMethod: () => adapter.getAllJobs(),
         refetchDependencies: [adapter],
         isAdapterCalledOnMount: false
     });
+    // hooks
+    const deleteJob = () => {
```

Comment:
shouldn't this take a job id at least to know which job to delete?

Reply:
It will. i just didn't include it because i'm not using the parameter for the setup and lint will get mad